orchagent/qosorch.cpp

```diff
@@ -1329,7 +1329,7 @@ task_process_status QosOrch::handlePortQosMapTable(Consumer& consumer)
 
         if (pfc_enable)
         {
-            if (!gPortsOrch->setPortPfc(port.m_port_id, pfc_enable))
+            if (!gPortsOrch->setPortPfc(port.m_port_id, pfc_enable, pfc_enable))
```

Comment:
the pfc_enable is on/off? if this is the case all other places are mask. can you please elaborate on that?

Reply:
Yes, it is according to requirements, "Asym PFC" is "on"/"off" port attribute for the user but SAI requires PFC value to be as bit vector.
https://github.com/Azure/SONiC/wiki/Asymmetric-PFC-High-Level-Design#11-functional-requirements